src/test/java/org/opentripplanner/updater/trip/RealtimeTestEnvironment.java

```diff
@@ -208,6 +208,14 @@ public TimetableSnapshot getTimetableSnapshot() {
     }
   }
 
+  public void commitTimetableSnapshot(boolean force) {
```

Comment:
The new test environment makes this unnecessary as the commit frequency is zero.